addons/mail/tests/discuss/test_ui.py

```diff
@@ -43,3 +43,8 @@ def test_04_meeting_view_tour(self):
             login="john",
         )
         self.start_tour(group_chat.invitation_url, "discuss.meeting_view_public_tour", login="john")
+
+    def test_05_create_channel_tour(self):
+        self.start_tour(
+            "odoo/discuss/discuss.channel", "create_channel_from_form_view", login="demo"
```

Comment:
`discuss.channel` in URL looks useless, no? `odoo/discuss` is all we need for discuss client action, rest of param is active_id but `discuss.channel` is no a valid active_id and the steps don't rely on loading initially with pre-selecting a conversation.